### PR TITLE
Update GettingStarted.md

### PR DESCRIPTION
**Pull request by dmhalejr**
## Proposed changes

- Add NVM blurb in `Taking the first steps` so those who don't have Node.js installed don't just install a static version the first time around
- Add PowerShell instructions to get `geckodriver` zip file downloaded for Windows users sake
- Add simple instructions to geckodriver installed quickly for Windows users (Chocolatey)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (clarifying statement and added documentation)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -39,6 +39,16 @@ $ curl -L https://github.com/mozilla/geckodriver/releases/download/v0.21.0/gecko
...
 Note: Other geckodriver releases are available [here](https://github.com/mozilla/geckodriver/releases). In order to automate other browser you need to run different drivers. You can find a list with all drivers in the [awesome-selenium](https://github.com/christian-bromann/awesome-selenium#driver) readme.
 
+Windows 64 bit (Powershell)
+
+```sh
+# Use geckodriver-v0.21.0-win32.zip for 32 bit Windows
+$url = "https://github.com/mozilla/geckodriver/releases/download/v0.21.0/geckodriver-v0.21.0-win64.zip"
+$output = "geckodriver.zip" # will drop into current directory unless defined otherwise
+
+Invoke-WebRequest -Uri $url -OutFile $output
```
Comment (christian-bromann) [inline docs/GettingStarted.md]: The windows example should be above the note in line 40, don't see a reason to separate it.

Comment (christian-bromann) [reply docs/GettingStarted.md]: maybe we should just use the geckodriver NPM package https://www.npmjs.com/package/geckodriver#use-it-globally

**Comment by christian-bromann** [comment]
ping @dmhalejr

File: docs/GettingStarted.md (hunk 1 above)
Comment (dmhalejr) [reply docs/GettingStarted.md]: Whoops yeah saw the change. Moved the Windows instructions before the note.

Hadn't used the geckodriver NPM package before but tried it out. I'm not opposed to doing that but did find the geckodriver on the Chocolatey package system. If you are not familiar just think homebrew but for Windows users and installed that without a hitch. 

Let me know if you'd me to pivot away from my simple and advanced instructions to something else! Thanks for the ping!

**Pull request merged**